src/ansys/fluent/core/quantity.py

```diff
@@ -333,6 +338,12 @@ def _compute_multipliers(self, unit_str, power):
                     _UnitsTable.derived_units[unit_str], term_power
                 )
 
+    def _compute_offset(self, unit_str):
+        if unit_str == "C" or unit_str == "C^1":
```

Comment:
or `if unit_str in ("C", "C^1")`

Later the logic will have to be less tied to the specific units, and even beyond that, to the string representation of those units.

Reply:
Yes, the upcoding code is already updated.